modules/apps/foundation/password-policies-admin/password-policies-admin-web/bnd.bnd

```diff
@@ -1,6 +1,8 @@
 Bundle-Name: Liferay Password Policies Admin Web
 Bundle-SymbolicName: com.liferay.password.policies.admin.web
 Bundle-Version: 1.0.0
+Include-Resource: @com.liferay.portal.impl-*.jar!/com/liferay/portal/model/PasswordPolicyImpl.class,\
+
```

Comment:
Hey @drewbrokke,

I don't think this is a good practice. 
@rotty3000 Do you think this is acceptable in order to be able to do new PasswordPolicyImpl() in the JSP for the form?
If not, one solution I can think of is to create a custom empty impl of PasswordPolicy. Can you think of something better?

@drewbrokke if Ray happens to think this is acceptable go ahead and send the pull to Brian.